src/ir_Midea.cpp

```diff
@@ -97,22 +95,22 @@ IRMideaAC::IRMideaAC(const uint16_t pin, const bool inverted,
     : _irsend(pin, inverted, use_modulation) { this->stateReset(); }
 
 /// Reset the state of the remote to a known good state/sequence.
-void IRMideaAC::stateReset(void) {
+void IRMideaAC::stateReset() {
   // Power On, Mode Auto, Fan Auto, Temp = 25C/77F
-  remote_state = 0xA1826FFFFF62;
+  _.remote_state = 0xA1826FFFFF62;
   _SwingVToggle = false;
   _EconoToggle = false;
 }
 
 /// Set up hardware to be able to send a message.
-void IRMideaAC::begin(void) { _irsend.begin(); }
+void IRMideaAC::begin() { _irsend.begin(); }
 
 #if SEND_MIDEA
 /// Send the current internal state as an IR message.
 /// @param[in] repeat Nr. of times the message will be repeated.
 void IRMideaAC::send(const uint16_t repeat) {
-  this->checksum();  // Ensure correct checksum before sending.
-  _irsend.sendMidea(remote_state, kMideaBits, repeat);
+  checksum();  // Ensure correct checksum before sending.
+  _irsend.sendMidea(_.remote_state, kMideaBits, repeat);
```

Comment:
Change to getraw call, and maybe move down the checksum comment inside getraw.

Reply:
thank you